apps/web/src/components/base-org/shared/TopNavigation/index.tsx

```diff
@@ -92,6 +93,13 @@ const links: TopNavigationLink[] = [
 ];
 
 export default function TopNavigation() {
+  const pathname = usePathname();
+  const showGasDropdownAndConnectWallet = ![
```

Comment:
Cool, I'd also extract this array to a named const out of this component but I think it barely matters.